scripts/templates/android/src/main.cpp

```diff
@@ -1,21 +1,19 @@
-#include "ofMain.h"
+#include "ofAppRunner.h"
 #include "ofApp.h"
 
 int main(){
-	ofSetupOpenGL(1024,768, OF_WINDOW);			// <-------- setup the GL context
-
-	// this kicks off the running of my app
-	// can be OF_WINDOW or OF_FULLSCREEN
-	// pass in width and height too:
-	ofRunApp( new ofApp() );
-	return 0;
+	// ofSetupOpenGL sets up a default OpenGL window for ofApp. The initial
```

Comment:
So should we be using the ofSetupOpenGL function here or should we be setting up the windows?  Or are you just suggesting removing the comment?